sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapExamples.cs

```diff
@@ -123,5 +123,124 @@ public void InterestRateSwapPortfolioCashFlowsExample(ModelSelection.ModelEnum m
             var irs = InstrumentExamples.CreateExampleInterestRateSwap("Vanilla");
             CallLusidGetPortfolioCashFlowsEndpoint(irs, model);
         }
+
+        [TestCase(ModelSelection.ModelEnum.ConstantTimeValueOfMoney)]
+        [TestCase(ModelSelection.ModelEnum.Discounting)]
+        public void LifeCycleManagementForInterestRateSwap(ModelSelection.ModelEnum model)
+        {
+            // CREATE an InterestRateSwap
+            var interestRateSwap = InstrumentExamples.CreateExampleInterestRateSwap(type: "Vanilla");
+            // Pick out the currency for future reference.
+            var fixedLeg = (FixedLeg) interestRateSwap.Legs
+                .First(leg => leg.InstrumentType == LusidInstrument.InstrumentTypeEnum.FixedLeg);
+            var currency = fixedLeg.LegDefinition.Conventions.Currency;
+
+            // CREATE wide enough window to pick up all cashflows associated to the InterestRateSwap
+            var windowStart = interestRateSwap.StartDate.AddMonths(-1);
+            var windowEnd = interestRateSwap.MaturityDate.AddMonths(1);
+
+            // CREATE portfolio and add instrument to the portfolio
+            var scope = Guid.NewGuid().ToString();
+            var (instrumentID, portfolioCode) = CreatePortfolioAndInstrument(scope, interestRateSwap);
+
+            // Populate stores with required market data.
+            CreateAndUpsertMarketDataToLusid(scope, model, interestRateSwap);
+
+            // CREATE recipe to price the portfolio with
+            var recipeCode = CreateAndUpsertRecipe(scope, model, windowValuationOnInstrumentStartEnd: true);
+
+            // We expect that the PV of the InterestRateSwap should be zero after the maturity.
+            // CREATE valuation request for this portfolio consisting of the InterestRateSwap,
+            // with valuation dates a few days before, day of and a few days after the instrument expiration.
+            var valuationRequest = TestDataUtilities.CreateValuationRequest(
+                scope,
+                portfolioCode,
+                recipeCode,
+                effectiveAt: interestRateSwap.MaturityDate.AddDays(2).AddMilliseconds(1),
+                effectiveFrom: interestRateSwap.MaturityDate.AddDays(-2).AddMilliseconds(1)); // TODO ANA-910: should I mention the millisecond issue?
```

Comment:
I think we should remove the comment and just the existence of the jira is sufficient.
Keeping it public doesn't look good and we may forget to remove when it is done.